src/newsletterCommon/sharedComponents.tsx

```diff
@@ -0,0 +1,236 @@
+import React, { useState } from 'react';
+import { css, ThemeProvider, keyframes } from '@emotion/react';
+import { Button, buttonThemeBrandAlt, SvgClock } from '@guardian/source-react-components';
+import type { TrackClick } from '../utils/tracking';
+
+import { news, neutral, body, textSans } from '@guardian/source-foundations';
+
+// Newsletter CTA button
+// -------------------------------------------------------
```

Comment:
At some point I think it will make sense to separate this, alongside the other 2 components defined in this file, into their own file

Reply:
Until then, what do you think about renaming this to `index.tsx`? Since this is the entirety of the common newsletter code right now. That would also give us nice imports like:

```typescript
import { CTA } from './newsletterCommon';

Reply:
Happy to do this

Reply:
Done